src/frontend-samples/query-volume-app/VolumeQueryApp.tsx

```diff
@@ -103,7 +103,8 @@ export class VolumeQueryApp implements SampleApp {
       return { className: element.className.replace(".", ":"), id: element.id };
     });
 
-    /* Sorting elements into packages. Instead of one 3000 length array I will have 3 x 1000 lenght arrays */
+    /* Breaking one array of 3000 keys into 3 arrays with 1000 keys each*/
```

Comment:
This comment still implies that there are always 3000 keys.  But that is not true.  The number of physical elements in every iModel is different.  The comment should say something like: "Break up the potential large array into smaller arrays with a maximum of 1000 keys each.  For example, if there are 3000 physical elements, this will create 3 arrays with 1000 keys each."